CONTRIBUTING.md

```diff
@@ -1,108 +1,117 @@
 # How to Contribute
- We welcome all contributions, suggestions, and feedback, so please do not hesitate to reach out!
 
- 
- We'd love to accept your patches and contributions to this project. There are just a few small guidelines you need to follow.
+We welcome all contributions, suggestions, and feedback, so please do not hesitate to reach out!
+
+We'd love to accept your patches and contributions to this project. There are just a few small guidelines you need to follow.
 
 ## Contributor License Agreement
 
- Contributions to this project must be accompanied by a Contributor License
- Agreement. You (or your employer) retain the copyright to your contribution;
- this simply gives us permission to use and redistribute your contributions as
- part of the project. Head over to <https://cla.developers.google.com/> to see your current agreements on file or to sign a new one.
+Contributions to this project must be accompanied by a Contributor License
+Agreement. You (or your employer) retain the copyright to your contribution;
+this simply gives us permission to use and redistribute your contributions as
+part of the project. Head over to <https://cla.developers.google.com/> to see your current agreements on file or to sign a new one.
 
- You generally only need to submit a CLA once, so if you've already submitted one
- (even if it was for a different project), you probably don't need to do it
- again.
+You generally only need to submit a CLA once, so if you've already submitted one
+(even if it was for a different project), you probably don't need to do it
+again.
 
 ## Code reviews
 
- All submissions, including submissions by project members, require review. We use GitHub pull requests for this purpose. Consult[GitHub Help](https://help.github.com/articles/about-pull-requests/) for more information on using pull requests.
+All submissions, including submissions by project members, require review. We use GitHub pull requests for this purpose. Consult[GitHub Help](https://help.github.com/articles/about-pull-requests/) for more information on using pull requests.
 
 ## Community Guidelines
 
- This project follows[Google's Open Source Community Guidelines](https://opensource.google.com/conduct/).
+This project follows[Google's Open Source Community Guidelines](https://opensource.google.com/conduct/).
```

Comment:
We follow the CNCF Code of Conduct, which is already referenced below. Are there specific items in the Google Community Guidelines we want to apply in addition? If not, would suggest simply referencing the CNCF CoC.